iPayments.js

```diff
@@ -0,0 +1,145 @@
+let libPrefix = "iPayments" 
+var price_api_url = "https://api.i-payments.site/check/price" 
+var build_api_url = "https://api.i-payments.site/v2/build" 
+var balance_api_url = "https://api.i-payments.site/v2/balance" 
+var deposit_api_url = "https://api.i-payments.site/v2/deposit" 
+var transfer_api_url = "https://api.i-payments.site/v2/transfer" 
+ 
+function setBashKey(key){ 
+Bot.setProperty(libPrefix + "bashkey", key, "string"); 
```

Comment:
no formatting

Reply:
It's Current Bot's set Property
And Check Coinpayments Lib too